luminaire/model/window_density.py

```diff
@@ -345,9 +330,25 @@ def _get_model(self, input_df=None, window_length=None, value_column=None, detre
                         opt_timestamp = timestamp
                         current_min_timedelta = temp_timedelta
 
-            if past_model and len(past_model._params['PastAnomalyScores'][opt_timestamp]) >= 10:
-                past_anomaly_scores = np.concatenate([past_model._params['PastAnomalyScores'][opt_timestamp][:10]
+                past_anomaly_scores = np.concatenate([past_model._params['PastAnomalyScores'][opt_timestamp][
+                                                      int(len(past_anomaly_scores) / 4): -int(len(past_anomaly_scores) / 4)]
```

Comment:
Propose to move `4` to constant variable with name or kwargs of function, as not it's hard to understand why is 4 and not something else.